### ISSUE TITLE
Python file for testing

### ISSUE THREAD

**Issue by piyushmishra12** [NJACKWinterOfCode/Fake-News-Detector#15]
Separate python file for testing the model.

**Comment by MVenkata-Rathnam** [comment]
which file should we use for testing? test.csv, data.csv or fake.csv

**Comment by piyushmishra12** [comment]
Initially I had only kept test.csv for testing. Let's keep it that way. If I need it later, I will add more data.

**Comment by MVenkata-Rathnam** [comment]
Ok.

**Comment by MVenkata-Rathnam** [comment]
can we modify the encode_data.py file in such a way that it can be reused for encoding the test data also but without saving it in a file?

**Comment by piyushmishra12** [comment]
Sure! But encoded training data should be stored in a file. Are you asking for suggestions or do you have a plan in mind?

**Comment by MVenkata-Rathnam** [comment]
Ok.  I was thinking not to store the encoding since the test file may get changed over time or even we may want to test it using a different file in the future.  But this is fine.

**Comment by piyushmishra12** [comment]
Okay seems fine. So you're saying we shouldn't store training data in a file? Won't that cause increased time in training (everytime we run the file it'll have to encode the data)?
If you want, you can go ahead with your plans for the test set so that when we change it, we encounter less problems. I hope I'm understanding your query properly?

**Comment by MVenkata-Rathnam** [comment]
No, generally I think training data won't change.  But since for testing you also collected some data as an issue I thought you would be adding it in future.

**Comment by MVenkata-Rathnam** [comment]
But no problem.  I will store the encoding for test file also.  Latter if we modify it we can see.

**Comment by piyushmishra12** [comment]
Cool
